docs/_documentations/remotedeploy-existingkeycloak.md

```diff
@@ -116,6 +114,7 @@ This command performs the following actions:
 - Creates an initial Codewind user called `developer` with password `passw0rd`.
 - Uses the ingress appropriate to the deployment environment.
 - Adds the Keycloak admin credentials of `admin` with password `passw0rd`.  
```

Comment:
The kadminuser and kadminpass dont so much "add" an admin user to keycloak (that is done separately when Keycloak is first installed).  Here, kadminuser and kadminpass are required by CWCTL to login to the already existing Keycloak in order to configure it with the realm/client/developer user items.

Reply:
Updated sentence to make it clear we're not setting the credentials here.